src/utils/monaco.ts

```diff
@@ -44,7 +45,8 @@ const schemas = [
 
 export default function setupMonaco() {
   Monaco.languages.json.jsonDefaults.setDiagnosticsOptions({
-    allowComments: false,
+    comments: 'warning',
+    trailingCommas: 'warning',
```

Comment:
Smart. So we don't even have to do any extra work to get warning.

Reply:
Yeah this way users get a warning in the editor, but it doesn't prevent rendering. I just realized that these warnings don't show up in the logs. Is there way to do that/should we do that?

Reply:
It would be nice. 

There must be a way to get the Monaco warning and show the in the console. 

We could probably replace some of the custom json schema validation if we figure this out.

Reply:
Yep. The biggest one is JSON schema validation. And because we use different validators, the error messages are also not very consistent.

Reply:
It would be awesome if you could look into this. If you feel it's too much of a change, we can also do it as a follow up pull request and merge this one first.

Reply:
I looked into it, and I think this would require hooking into the `onDidChangeMarkers` callback, which would require changing the React wrapper around Monaco or switching to the other Monaco editor wrapper @monaco-editor/react.

So I think it should be a separate PR.

Reply:
Okay. I own the monaco component so if you send a pull request, I can review and release. Let's do this in a follow up.